spyder/plugins/pylint/rule_ignorer.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+# Standard library imports
+import re
+
+
+class RuleIgnorer(object):
+    """Class for insert lint rules ignores automatically."""
+
+    __base_ignore_comment = '# pylint: disable='
+
+    def __init__(self, code_editor):
+        """Initialize and Add code_editor to the variable."""
+        self.__code_editor = code_editor
+
+    def __is_function_or_class_definition(self):
+        text_at_line = self.__code_editor.get_text('sol', 'eol').strip()
+        return text_at_line.startswith('def') or text_at_line.startswith(
+            'class')
+
+    def __exist_other_ignored_rules(self, line_number):
+        self.__code_editor.go_to_line(line_number)
+        return self.__code_editor.get_text('sol', 'eol').strip().startswith(
+            self.__base_ignore_comment)
+
+    def __calculate_line_number_for_comment(self, line_number):
+        self.__code_editor.go_to_line(line_number)
+        if self.__is_function_or_class_definition():
+            return line_number + 1
+        else:
+            if self.__exist_other_ignored_rules(line_number - 1):
+                return line_number - 1
+            else:
+                return line_number
+
+    def __add_new_rule(self, rule_id):
+        indentation = self.__get_text_indentation(
+            self.__code_editor.get_text('sol', 'eol'))
+        ignore_comment = \
+            f"{indentation}{self.__base_ignore_comment} {rule_id}\n"
```

Comment:
```suggestion
        ignore_comment = (
            f"{indentation}{self.__base_ignore_comment} {rule_id}\n")
```

Prefer `(` to `\` for continuation